chipy_org/apps/meetings/models.py

```diff
@@ -125,7 +132,14 @@ def get_absolute_url(self):
     def meetup_url(self):
         return "https://www.meetup.com/_ChiPy_/events/{}/".format(self.meetup_id)
 
-
+    @property 
+    def title(self):
+        if self.custom_title:
+            return self.custom_title
+        if self.meeting_type and self.meeting_type.default_title:
+            return self.meeting_type.default_title
+        return "" # return an empty string if neither title is defined
+    
```

Comment:
If you want, you could make this final return the default for the main meeting as not custom title and no meeting type is a main meeting with a default title.  I'd put whatever is in the template now.

Reply:
Sorry, could you please explain this? I'm not following.

> If you want, you could make this final return the default for the main meeting as not custom title and no meeting type is a main meeting with a default title. I'd put whatever is in the template now.